server/routes/cards.js

```diff
@@ -3,19 +3,65 @@
 const express = require('express');
 const router = express.Router();
 const Card = require('../database/models/Card');
+const Word = require('../database/models/Word');
+const knex = require('../database/knex');
 
-router.route('/')
+router
+  .route('/')
   // fetches all cards
   .get((req, res) => {
     new Card()
       .fetchAll({ withRelated: ['users', 'words', 'card_themes', 'created_by'] })
       .then((results) => {
-        return res.send(results.toJSON());
+        return res.json(results.toJSON());
```

Comment:
shouldn't need toJSON() since you have res.json(results)

Reply:
changed.